### ISSUE TITLE
Version 1.28.1 on MacOS 10.14 crashed on page reload and deleted all my tasks

### ISSUE THREAD

**Issue by cbertelli** [minbrowser/min#2290]
1.28.1
macOS 10.14

Choosing page reload

Crashed and all my tasks are gone

I don't know how to reproduce it, I hope it will not happen again. I don't know how to recover all my precious tasks

It let a huge ~/Library/Application Support/Min/ folder. I don't know why.

**Comment by PalmerAL** [comment]
Sorry to hear that!

Did you use the cmd+r shortcut, or the developer > reload page option?

**Comment by cbertelli** [comment]
Yes, this is the real issue. Min didn't refresh the page pressing F5, nor
cmd-F5, nor cmd-r.
Maybe it's an Electron issue. Anyway I would like to recover my old tasks,
I would very much appreciate it. I was following several projects at
the same time and Min is great for ultra-busy people.
c

**Comment by Kepler-69c** [comment]
Were you able to recover your tasks?
A large `~/Library/Application Support/Min/` is normal, if you have many tabs/tasks open, because that's where the browser stores the browsing data and caches the websites. This means that your tasks are still around, but somehow not displayed by the browser. I can't tell you what the problem is without more information, thought.

- Were you able to reproduce the bug?
- Do you know the website where it happened?
- What is the size of the `~/Library/Application Support/Min/` folder, how many open tabs did you have?
